cmake/configs/posix_sitl_default.cmake

```diff
@@ -97,7 +97,7 @@ set(config_extra_builtin_cmds
 	)
 
 set(config_sitl_rcS_dir
-	posix-configs/SITL/init/lpe
+	posix-configs/SITL/init/ekf2
 	CACHE FILEPATH "init script dir for sitl"
```

Comment:
@LorenzMeier  This will have no impact on the SITL CI which uses ros. You would need to modify the rostest script here. Would be great if we could get both estimators working though. I haven't seen any flakiness regarding altitude init, so might be something recent.

https://github.com/PX4/Firmware/blob/master/launch/mavros_posix_tests_iris.launch#L7